Maskbook/Wallet/Model/WalletDisplayBlockChainType.swift

```diff
@@ -157,4 +157,37 @@ extension WalletDisplayBlockChainType: Equatable {
             }
         }
     }
+    
+    var bottomBgColor: CGColor {
+        switch self {
+        case .all: return Asset.Colors.AccountCard.Chains.all3.color.cgColor
+            
+        case .blockchain(let blockchain):
+            switch blockchain.rawValue {
```

Comment:
As `BlockChainNetwork` conforms to `BlockChainNetworkRepresentable`, which is inherited from `RawRepresentable`, we can use `blockchain` instead of `blockchain.rawValue`.